src/Storages/ObjectStorage/DataLakes/Iceberg/IcebergWrites.cpp

```diff
@@ -268,6 +275,7 @@ FileNamesGenerator & FileNamesGenerator::operator=(const FileNamesGenerator & ot
     storage_dir = other.storage_dir;
     use_uuid_in_metadata = other.use_uuid_in_metadata;
     compression_method = other.compression_method;
+    format_name = other.format_name;
```

Comment:
Also for `FileNamesGenerator::FileNamesGenerator(const FileNamesGenerator & other)`?